emmo/.htaccess

```diff
@@ -0,0 +1,127 @@
+RewriteRule ^domain/(domain-)?([^/]+)/([0-9][^/]+)/([^0-9].+)/?$ https://raw.githubusercontent.com/emmo-repo/$1$2/$3/$4.ttl [R=303,NE,L]
+
+
+
+##########################################################################
+##  Redirections for EMMO
+##########################################################################
+
+# Rule 7: https://w3id.org/emmo
+#
+# If the request appears coming from a browser, redirect to the html documentation
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml
+RewriteRule ^$ https://emmo-repo.github.io/emmo.html [R=303,NE,L]
+#
+# ...otherwise redirect to inferred/squashed ontology
+RewriteRule ^emmo-inferred$ https://emmo-repo.github.io/emmo-inferred.ttl [R=303,NE,L]
+RewriteRule ^$ https://emmo-repo.github.io/emmo.ttl [R=303,NE,L]
+
+
+# Rule 8: https://w3id.org/emmo/
+# Alias:  https://w3id.org/emmo/latest
+# Alias:  https://w3id.org/emmo/source
+#
+# emmo.ttl file in the root of the master branch
+RewriteRule ^/$ https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl [R=303,NE,L]
+RewriteRule ^/latest/?$ https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl [R=303,NE,L]
+RewriteRule ^/source/?$ https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl [R=303,NE,L]
+
+
+# Rule 9: https://w3id.org/emmo/{VERSION}
+#
+# If the request appears coming from a browser, redirect to the html documentation of given version
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml
+RewriteRule ^([0-9][^/]+)$ https://emmo-repo.github.io/versions/$1/emmo.html [R=303,NE,L]
+#
+# ...otherwise redirect to inferred/squashed ontology
+RewriteRule ^([0-9][^/]+)-inferred$ https://emmo-repo.github.io/versions/$1/emmo-inferred.ttl [R=303,NE,L]
+RewriteRule ^([0-9][^/]+)$ https://emmo-repo.github.io/versions/$1/emmo.ttl [R=303,NE,L]
+
+
+
+# Rule 10: https://w3id.org/emmo/{VERSION}/
+# Alias:   https://w3id.org/emmo/{VERSION}/source
+#
+# emmo.ttl file in the root of branch for the given version
+RewriteRule ^([0-9][^/]+)/$ https://raw.githubusercontent.com/emmo-repo/EMMO/$1/emmo.ttl [R=303,NE,L]
+RewriteRule ^([0-9][^/]+)/source/?$ https://raw.githubusercontent.com/emmo-repo/EMMO/$1/emmo.ttl [R=303,NE,L]
+
+
+# Rule 11: https://w3id.org/emmo/{PATH}/{MODULE}
+#
+# Turtle file for given module of EMMO
+RewriteRule ^([^0-9].+)/?$ https://raw.githubusercontent.com/emmo-repo/EMMO/$1.ttl [R=303,NE,L]
+
+
+# Rule 12: https://w3id.org/emmo/{VERSION}/{PATH}/{MODULE}
+#
+# Turtle file for given version and module of EMMO
+RewriteRule ^([0-9][^/]+)/([^0-9].+)/?$ https://raw.githubusercontent.com/emmo-repo/EMMO/$1/$2.ttl [R=303,NE,L]
+
+
+
+##########################################################################
+##  Redirections for http://emmo.info/
+##########################################################################
+
+# Redirect to emmo.info/
+ RewriteRule	^$ http://emmo.info/ [R=303,L]
```

Comment:
Looks like this is already defined above.

Reply:
You are right. Removed it.